server/conf.go

```diff
@@ -577,6 +581,68 @@ func (conf *Configuration) redisTLSConfig() (*tls.Config, error) {
 	return tlsConfig, nil
 }
 
+func readSdJwtIssuerPivKeys(dir string) (map[irma.IssuerIdentifier]*ecdsa.PrivateKey, error) {
+	entries, err := os.ReadDir(dir)
```

Comment:
Suggestion: use `filepath.Glob(filepath.Join(dir, "*.pem:"))` to filter the entries before looping over the entries.
That will make the check in the loop redundant.